db/schema.rb

```diff
@@ -102,6 +102,9 @@
     t.string "password_salt"
     t.boolean "external", default: false, null: false
     t.boolean "blocked", default: false, null: false
+    t.string "provider"
+    t.string "uid"
```

Comment:
Where does it come from?

Reply:
It's from `rake db:migrate`. I think it's old not applied changes

Reply:
It's seems to be some stuff from devise/omniauth: https://github.com/hydra-billing/homs/blob/f88f7264f2b74754277ed06e4b0f6279565fbb81/config/initializers/devise.rb#L234-L237

We don't use omniauth, so I'll remove it.